src/features/HelpAndKnowledge/Articles/TwoPagerMeeting/table.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Trans } from 'react-i18next';
+import { Table as TrussTable } from '@trussworks/react-uswds';
+
+import { tArray, tObject } from 'utils/translation';
+
+import { covertToLowercaseAndDashes } from '..';
+
+type ContentTypes = {
+  criteria: string;
+  description: string;
+  score: string;
+};
+
+const ModelSectionCriteriaTable = () => {
+  const headers = tArray('twoPageMeeting:reviewMeeting.criteria.table.header');
+
+  const content = tObject(
+    'twoPageMeeting:reviewMeeting.criteria.table.content'
+  ) as ContentTypes[];
+
+  return (
+    <TrussTable bordered={false} fullWidth fixed>
+      <thead>
```

Comment:
Thanks for breaking this table out.  Let's sync when you return about a strategy for our PRs to be able to use all of these.  It might make sense to quickly break out our shared components and get a quick review on these so we can both incorporate them